app/js/components/tabs/tabcomponents/encounterComponent.jsx

```diff
@@ -103,8 +103,8 @@ class EncounterComponent extends Component {
 
       const searchParams = this.getFormValues(fields);
       this.props.search(searchParams).then(results => {
-          const allEncounterTypes = results.rows;
-          this.props.addToHistory(results.searchDescription, allEncounterTypes.length, searchParams);
+          const allEncounterTypes = results.rows || [];
+          this.props.addToHistory(results.searchDescription, allEncounterTypes, searchParams);
```

Comment:
Do we need to pass searchParams again since we won't be making searches anymore? The data is already stored in sessionStorage